vnext/Shared/OInstance.cpp

```diff
@@ -579,6 +580,10 @@ void InstanceImpl::loadBundleInternal(std::string &&jsBundleRelativePath, bool s
       m_innerInstance->loadScriptFromString(std::move(bundleString), jsBundleRelativePath, synchronously);
 #endif
 
+#if defined(_CHAKRACORE_H_)
+    } catch (const facebook::react::ChakraJSException &e) {
+      m_devSettings->errorCallback(std::string{e.what()} + "\r\n" + e.getStack());
```

Comment:
Do we really want the carriage return?

Reply:
Let me check if the current message already include the return. If it doesn't, then we should have the return to make the message easier to read.

Reply:
I mean the "\r" specifically. I.e. CRLF vs LF

Reply:
Because this repo is for react on Windows, I figured that I should use Windows new line character.